docs/tutorial/exomiser-tutorial.md

```diff
@@ -0,0 +1,680 @@
+## Tutorial: an introduction to Exomiser
```

Comment:
If you like you can include a [maturity indicator badge](https://oboacademy.github.io/obook/reference/obook-maturity-indicator/), see for example

https://oboacademy.github.io/obook/lesson/contributing-to-obo-ontologies/

Reply:
Yes, this will be perfect, I'll add it!